locale/fr/get-involved/index.md

```diff
@@ -3,44 +3,48 @@ title: Comment s'impliquer ?
 layout: contribute.hbs
 ---
 
-# Nous encourageons toute forme de contributions de la part de la communauté 
+# Nous encourageons toutes formes de contribution de la part de la communauté 
 
-La communauté Node.js est vaste, inclusive et heureuse de permettre à autant 
+La communauté Node.js est vaste, inclusive et entousiaste à l'idée de permettre à autant 
 d'utilisateurs de contribuer de toutes les manières possibles. Si vous souhaitez
 [signaler un problème](https://github.com/nodejs/node/issues), aider à documenter
 ou contribuer au code du projet, vous êtes au bon endroit. Explorez les ressources 
-de notre communauté pour trouver comment vous pouvez aider:
+de notre communauté pour trouver comment vous pouvez aider&nbps;:
 
 <hr>
 
-## Discussion de la communauté
+## Discussions de la communauté
 
-- La [liste des tickets GitHub](https://github.com/nodejs/node/issues) est le bon endroit pour discuter des fonctionnalités coeurs de Node.js.
+- La [liste des tickets GitHub](https://github.com/nodejs/node/issues) est le bon endroit pour discuter des fonctionnalités principales de Node.js.
```

Comment:
I'm not a big fan of "principale" here. The original term is probably "core" and refers to any features shipped directly with Node.js itself.

Reply:
Okay. [Python indeed this term too](https://docs.python.org/3/library/intro.html?highlight=core) (wrong hunch on my side).